.travis.yml

```diff
@@ -0,0 +1,38 @@
+env:
+  global:
+    - RUST_BACKTRACE=1
+    - PATH=$PATH:$HOME/.cargo/bin
+os:
+  - linux
+  - osx
+language: rust
+rust:
+  - stable
+  - nightly
+matrix:
+  allow_failures:
+    - rust: nightly
+sudo: false
+branches:
+  only:
+    - master
+    - dev
+cache:
+  cargo: true
+before_script:
+  - (which cargo-install-update && cargo install-update cargo-update) || cargo install cargo-update
+  - (which rustfmt && cargo install-update rustfmt) || cargo install rustfmt
+  - (which cargo-prune && cargo install-update cargo-prune) || cargo install cargo-prune
+script:
+  - if [ "${TRAVIS_RUST_VERSION}" = stable ]; then
+      (
+        set -x;
+        cargo fmt -- --write-mode=diff &&
+        cargo rustc --verbose --release -- --test -Zno-trans &&
+        cargo test  --verbose --release --features use-mock-routing
```

Comment:
we don't have this feature here so it's useless, infact it will generate an error in travis.
so after cargo fmt all we need is:
`cargo test  --verbose --release`
and _not_ even `cargo rustc --verbose --release -- --test -Zno-trans &&` line